system/integrations/cti/includes/troubleshooting.include.rst

```diff
@@ -0,0 +1,10 @@
+Troubleshooting
+---------------
+
+My Phone page stays blank, signalling does work...
+   If you've made sure that signalling works (check Recent logs) and your
+   Caller Log still stays empty, please ensure that you only configured one
+   CTI integration version. Specifically defining more than agent mappings on
+   different integration versions will be the issue.
```

Comment:
"Specifically defining more than agent mappings on different integration versions will be the issue."

Makes no sense to me. Maybe you wanted to say?
"Specifically defining more than one agent mapping on the different integration variants will be the issue."

If I didn't catch the intent, please rephrase.